worlds/terraria/docs/setup_en.md

```diff
@@ -7,16 +7,28 @@ and [TModLoader](https://store.steampowered.com/app/1281930/tModLoader/) on Stea
 
 ## Installing the Archipelago Mod
 
-Subscribe to [the mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2922217554) on Steam.
+1. Subscribe to [the mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2922217554) on Steam
+2. Open TModLoader
+3. Go to `Workshop > Manage Mods` and enable the Archipelago mod
+   - If TModLoader states that you need version 1.4.3, follow the following steps
+     1. Close TModLoader
+     2. Right-Click TModLoader in Steam and select `Properties`
+     3. Navigate to `Betas -> Beta Participation`
+     4. Select `1.4.3-legacy - Legacy - Stable tModLoader for Terraria 1.4.3`
+     5. Update TModLoader through Steam
+     6. Open TModLoader and navigate back to the **Manage Mods** menu
+4. TModLoader will say that it needs to refresh; exit this menu, and it will do this automatically
+5. Once TModLoader finishes loading, the Archipelago mod is finished installing and you can now 
+[connect to an Archipelago game](#joining-an-archipelago-game-in-terraria).
 
-This mod might not work with mods that significantly alter progression or vanilla features. It is
+This mod may not work with mods that significantly alter progression or vanilla features. It is
 highly recommended to use utility mods and features to speed up gameplay, such as:
 
 - Journey Mode
 - Ore Excavator
 - Magic Storage
 - Alchemist NPC Lite
-    - (May be used to break progression)
+    - (Possibly breaks progression)
```

Comment:
"May be used to break" is better, since it conveys that the progression-breaking is up to the player perform. The mod won't break progression unless the player uses it to do so.

Reply:
Change mostly reverted in e9183cb7cbe6d0d2a64eae3cd82785d0543d90bf  
Made change "May -> Can" to reinforce the idea